src/layouts/components/ContactUs/GeneralInfoSection.tsx

```diff
@@ -36,7 +36,7 @@ export const GeneralInfoSection = ({
       title="General Information"
       isInvalid={!!errors.agency_name || !!errors.feedback}
     >
-      <Editable.Section spacing="1.25rem">
+      <Editable.Section spacing="1.25rem" pb="1.25rem">
```

Comment:
is the `pb` only valid for `GeneralInfoSection`?

Reply:
Yup, the rule is "the padding should be larger if there is no action button at the end of the card".